### PR TITLE
feat: updating limitations of Reactors

### PR DESCRIPTION
**Pull request by armsteadj1**
## Description

- Updates the reactor limitations

## Testing required outside of automated testing?

- [ ] Not Applicable

### Screenshots (if appropriate):

image

## Rollback / Rollforward Procedure

- [ ] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: docs/api/reactors/reactors.mdx [hunk 1]
```diff
@@ -1306,6 +1304,14 @@ Errors generated from Reactors will be translated to the common Basis Theory Err
 | `tokens`      | _object_ | (Optional) [Token(s)](/docs/api/tokens#token-object) created from the `tokenize` block of the Reactor code [response](#reactor-code-response-object) |
 | `raw`         | _object_ | (Optional) Raw output returned from the Reactor                                                                                                                                               |
 
+### Limitations
+
+| Limit        | Note                                                                                                                                                                                   |
+| ------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Payload Size  | The maximum payload size to invoke a Reactor is 5 MB. You could pull in more data using the injected <code>BT</code> client from within the Reactor code. <a href="https://basistheory.com/contact">Please reach out</a> if you need help. |
```
Comment (djejaquino) [inline docs/api/reactors/reactors.mdx:1311]: > You could pull in more data using the injected <code>BT</code> client from within the Reactor code.

What do you mean? Can you give an example?

Comment (djejaquino) [inline docs/api/reactors/reactors.mdx:1311]:
```suggestion
| Payload Size  | The maximum payload size to invoke a Reactor is 5 MB. You could pull in more data using the injected <code>bt</code> client from within the Reactor code. <a href="https://basistheory.com/contact">Please reach out</a> if you need help. |
```

**Review by djejaquino — CHANGES_REQUESTED** [review]

File: docs/api/reactors/reactors.mdx (hunk 1 above)
Comment (armsteadj1) [reply docs/api/reactors/reactors.mdx:1311]: I just copied this from our message there for a few months, haha. I don't know how to explain more. I think removing that might be the best or trying to think of a way to make it more broad - I don't think many people have this problem and if they do I think we've really only seen people pulling in non-token data.

Thoughts

```suggestion
| Payload Size  | The maximum payload size to invoke a Reactor is 5 MB. <a href="https://basistheory.com/contact">Please reach out</a> if you need help using larger datasets. |
```

Comment (djejaquino) [reply docs/api/reactors/reactors.mdx]: Yea, let me make another suggestion to suit that better.

File: docs/api/reactors/reactors.mdx [hunk 2]
```diff
@@ -1306,6 +1304,14 @@ Errors generated from Reactors will be translated to the common Basis Theory Err
 | `tokens`      | _object_ | (Optional) [Token(s)](/docs/api/tokens#token-object) created from the `tokenize` block of the Reactor code [response](#reactor-code-response-object) |
 | `raw`         | _object_ | (Optional) Raw output returned from the Reactor                                                                                                                                               |
 
+### Limitations
+
+| Limit        | Note                                                                                                                                                                                   |
+| ------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Payload Size  | The maximum payload size to invoke a Reactor is 5 MB. You could pull in more data using the injected <code>bt</code> client from within the Reactor code. <a href="https://basistheory.com/contact">Please reach out</a> if you need help. |
```
Comment (djejaquino) [inline docs/api/reactors/reactors.mdx:1311]:
```suggestion
| Payload Size  | The maximum payload size to invoke a Reactor is 5 MB. <a href="https://basistheory.com/contact">Please reach out</a> if your use case requires more than that. |
```

**Review by djejaquino — CHANGES_REQUESTED** [review]

**Review by djejaquino — APPROVED** [review]

**Pull request merged**